### PR TITLE
[SNAP-1377] DIMAP discontinuity

### PR DESCRIPTION
**Pull request by SabineEmbacher**

### CODE REVIEW

**Review by marpet — CHANGES_REQUESTED** [review]
The tests are not yet fully working. Can you update this:
https://travis-ci.org/github/senbox-org/snap-engine/jobs/754336985#L4647

**Comment by SabineEmbacher** [comment]
Tests now are fixed.

File: snap-core/src/main/java/org/esa/snap/core/dataio/dimap/DimapHeaderWriter.java [hunk 1]
```diff
@@ -406,8 +406,10 @@ protected void writeTiePointGridElements(int indent) {
                 printLine(indent + 2, DimapProductConstants.TAG_TIE_POINT_OFFSET_Y, tiePointGrid.getOffsetY());
                 printLine(indent + 2, DimapProductConstants.TAG_TIE_POINT_STEP_X, tiePointGrid.getSubSamplingX());
                 printLine(indent + 2, DimapProductConstants.TAG_TIE_POINT_STEP_Y, tiePointGrid.getSubSamplingY());
-                final boolean cyclic = tiePointGrid.getDiscontinuity() != TiePointGrid.DISCONT_NONE;
-                printLine(indent + 2, DimapProductConstants.TAG_TIE_POINT_CYCLIC, cyclic);
+                final int discontinuity = tiePointGrid.getDiscontinuity();
+                final boolean cyclic = discontinuity != TiePointGrid.DISCONT_NONE;
+                final String[][] attributes = new String[][]{new String[]{DimapProductConstants.ATTRIB_TIE_POINT_DISCONTINUITY, "" + discontinuity}};
```
Comment (marpet) [inline snap-core/src/main/java/org/esa/snap/core/dataio/dimap/DimapHeaderWriter.java:411]: Can we replace 
`"" + discontinuity` 
by
`String.valueOf(discontinuity)`

**Review by marpet — COMMENTED** [review]
Just a small comment. After changed you can merge this.

**Pull request merged**